bellman/src/groth16/generator/assembly/eval.rs

```diff
@@ -0,0 +1,205 @@
+use ff::{Field, PrimeField};
+use group::{CurveProjective, Wnaf};
+use pairing::Engine;
+
+use crate::{multicore, arith};
```

Comment:
Good organisation of imports, but IMO it's better to have the module specific below and external libraries above

Reply:
I don’t understand the comment, the external imports are currently declared above. Can you clarify please?

Reply:
Resolved